WWDC/AppCoordinator.swift

```diff
@@ -257,15 +257,15 @@ final class AppCoordinator {
         // Initial app launch waits for all of these things to be loaded before dismissing the primary loading spinner
         // It may, however, delay the presentation of content on tabs that already have everything they need
 
-        let starupDependencies = Observable.combineLatest(storage.tracksObservable,
+        let startupDependencies = Observable.combineLatest(storage.tracksObservable,
```

Comment:
Pretty sure this typo was mine 🙈. I do this at work all the time, too!

Reply:
I have a colleague that does it all the time, so I'm used to fixing typos in code review :P